### ISSUE TITLE
Country Code Format

### ISSUE THREAD

**Issue by sdorazio** [openrtb/OpenRTB#15]
The OpenRTB 2.2 spec calls for ISO-3166-1 alpha-3 for the country code format. The majority of systems today use ISO-3166-1 alpha-2 for country codes (see http://en.wikipedia.org/wiki/ISO_3166-1).  I'd like to propose re-evaluating the format used for country codes, in favor of ISO-3166-1 alpha-2, as it is the standard for most systems. Popular GeoIP services such as MaxMind lack support for ISO-3166-1 alpha-3 codes. The alpha-3 format does not provide any additional value for server-to-server calls, and quickly causes incompatibility issues with other systems.

**Comment by opinali** [comment]
+1, speaking for Ad Exchange the alpha-3 codes are just yet another thing we need to translate since we only use alpha-2.

**Comment by cpats007** [comment]
Agreed - we also find the 3 code system a translation pain as our whole infrastructure uses the 2 code Country system. Why was this standard deemed better than using ISO-3166-1 alpha-2?
